### PR TITLE
Fix bug take method

### PR DESCRIPTION
**Pull request by vanyarock01**
[issue](https://github.com/tarantool/sharded-queue/issues/2)
Error when calling an undefined conditional variable.

### CODE REVIEW

File: sharded_queue/drivers/fifottl.lua [hunk 1]
```diff
@@ -26,7 +26,7 @@ local function is_expired(task)
     return (task[index.created] + task[index.ttl]) <= time.cur()
 end
 
-local fibers_info = {}
+local wc_table = {}
```
Comment (sharonovd) [inline sharded_queue/drivers/fifottl.lua:29]: What does wc_table stand for?

File: sharded_queue/drivers/fifottl.lua [hunk 2]
```diff
@@ -86,29 +86,22 @@ local function fiber_iteration(tube_name, processed)
...
         estimated = estimated > 0 and estimated or 0
-
-        local cond = fiber.cond()
-
-        fibers_info[tube_name] = {
-            cond = cond
-        }
-
-        cond:wait(estimated)
+        wc_table[tube_name].cond:wait(estimated)
```
Comment (sharonovd) [inline sharded_queue/drivers/fifottl.lua:89]: I'd check for nil anyway

File: sharded_queue/drivers/fifottl.lua (hunk 1 above)
Comment (vanyarock01) [reply sharded_queue/drivers/fifottl.lua:29]: Wait condition table

Comment (sharonovd) [reply sharded_queue/drivers/fifottl.lua:29]: This comment should habe been in the code then.
I suggest to rename variable to `wait_cond_map` then.

File: sharded_queue/drivers/fifottl.lua (hunk 2 above)
Comment (vanyarock01) [reply sharded_queue/drivers/fifottl.lua]: done

**Pull request merged**